ckan/authz.py

```diff
@@ -430,12 +430,15 @@ def auth_is_anon_user(context):
 
         See ckan/lib/base.py:232 for pylons context object logic
     '''
+    is_ip = lambda x: re.match('\d+\.\d+\.\d+\.\d+', x)
+    is_unknown_ip = lambda x: x == 'Unknown IP Address'
```

Comment:
This magic value comes from https://github.com/ckan/ckan/blob/master/ckan/lib/base.py#L261 so I'd encourage you to reference it in a comment.